adal/src/main/java/com/microsoft/aad/adal/AcquireTokenRequest.java

```diff
@@ -261,7 +261,7 @@ private AuthenticationResult tryAcquireTokenSilent(final AuthenticationRequest a
 
 
     private boolean shouldTrySilentFlow(final AuthenticationRequest authenticationRequest) {
-       return authenticationRequest.getPrompt() == PromptBehavior.Auto || authenticationRequest.isSilent();
+        return authenticationRequest.getPrompt() == PromptBehavior.Auto || authenticationRequest.isSilent();
```

Comment:
hm, can you kindly revert this? Each of us have the assigned files to clean up checkstyle changes, my worry is that this may have already been cleaned up.